charts/service-library/templates/_ingress.yaml

```diff
@@ -6,16 +6,15 @@ metadata:
   name: {{ .Release.Name }}
   labels: {{- include "service-library.labels" . | nindent 4 }}
   annotations:
-    kubernetes.io/ingress.class: alb
-    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}]'
-    {{- if eq .Values.ingress.visibility "public" }}
-    alb.ingress.kubernetes.io/group.name: {{ .Values.ingress.domain }}
-    alb.ingress.kubernetes.io/scheme: internet-facing
-    {{- else }}
-    alb.ingress.kubernetes.io/group.name: {{ .Values.ingress.domain }}-internal
-    alb.ingress.kubernetes.io/scheme: internal
-    {{- end }}
+    # this annotation is purely here to let external-dns-private-nginx find this ingress and create a record
```

Comment:
@paul-ktech I should have asked this before approving, but how do we distinguish between cluster, internal and internet facing now, given the ingress schemes have gone?